open-company/open-company-web
# Replace org placeholder avatar in onboard screen.

Card: https://trello.com/c/dbvLHXY9

Bug from Ryan: the upload org avatar during onboarding is not aligned, provided a new file.

To test:
- /sign-up
- enter email pswd
- enter first and last name
- [x] do you see the new image as org avatar? Good
